### ISSUE TITLE
Question for train.py

### ISSUE THREAD

**Issue by CeyaoZhang** [guosyjlu/DS-Agent#15]
Why does the validation data directly use `compute_metrics_for_classification(y_valid, y_valid_pred)` but the test data use `submit_predictions_for_test_set(y_test_pred)`? It seems that we can also use `compute_metrics_for_classification(y_test, y_test_pred)`.

**Comment by guosyjlu** [comment]
Yes, you are right. Both are okay. It's just a design to isolate validation and test in case somebody wants to save the prediction for test set.
